tests/semantic_analyser.cpp

```diff
@@ -4367,8 +4369,8 @@ stdin:1:1-10: ERROR: Only single iter attach point is allowed.
 iter:task,f:func_1 { 1 }
 ~~~~~~~~~
 )" });
-  test("iter:task* { }", Error{ R"(
-stdin:1:1-11: ERROR: iter probe type does not support wildcards
```

Comment:
Nit. I do prefer this older error message. Don't we have the probe string at the time of issuing this message and can therefore make the distinction between `iter:task,iter:task_vma` and `iter:task*` ?

Reply:
We have `ap->raw_input` so we could check if the current attach point has been expanded. Restored the original error message.